gnucash/gnome-utils/dialog-account.c

```diff
@@ -2206,14 +2206,13 @@ static void
 gnc_account_renumber_update_examples (RenumberDialog *data)
 {
     gchar *str;
-    gchar *prefix;
     gint   interval;
     gint   digits;
     unsigned int num_digits = 1;
 
     g_return_if_fail (data->num_children > 0);
 
-    prefix = gtk_editable_get_chars (GTK_EDITABLE(data->prefix), 0, -1);
+    const gchar *prefix = gtk_entry_get_text (GTK_ENTRY(data->prefix));
```

Comment:
`gtk_entry_get_text` [can return `NULL`](https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/gtk/gtkentrybuffer.c#L525) and `strlen` will crash if fed a `NULL` glibc and clang's libc, so the test check at line 2237 and elsewhere needs to be `if (prefix && strlen(prefix))` or better `if (prefix && *prefix)` to not count the chars for no good reason.